.drone.star

```diff
@@ -1996,7 +1999,7 @@ def middlewareService():
         "image": OC_OC_TEST_MIDDLEWARE,
         "environment": {
             "BACKEND_HOST": "https://ocis-server:9200",
-            "OCIS_REVA_DATA_ROOT": "/srv/app/tmp/ocis/storage/owncloud/",
```

Comment:
not related to this PR, but I'm surprised that we don't add `/data` here at the end for the middleware